generator/config/pipeline-operators.yaml

```diff
@@ -0,0 +1,92 @@
+- name: and
+  type: resolvesToBool
+  args:
+    - name: expressions
+      type: expression
+      isVariadic: true
+      variadicMin: 1
+- name: eq
```

Comment:
Not an issue here, but we should consider splitting this file into multiple smaller files in the future.